apps/payments/next/app/%5F_heartbeat__/route.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextResponse } from 'next/server';
+
+import { app } from '../nestapp/app';
+
+export const dynamic = 'force-dynamic';
```

Comment:
These are necessary to prevent NextJS from trying to build these as static pages.